### PR TITLE
goosies page + home page changes

### PR DESCRIPTION
**Pull request by travisfriesen**

### CODE REVIEW

File: src/utils/Routes.tsx [hunk 1]
```diff
@@ -54,4 +60,5 @@ export const Routes: IRoute[] = [
         path: '/MovieNight',
         component: (<MovieNight/>)
     }
+
```
Comment (lysackm) [inline src/utils/Routes.tsx:63]: Not a big deal but just remove the line

File: src/pages/Goosies/Goosies.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import { Text, HStack, Box, Icon, Button, Link } from "@chakra-ui/react";
+import { PageLayout } from "../../containers/PageLayout/PageLayout";
+import { EventTeaser } from "../../components/EventTeaser/EventTeaser";
+import { MdOutlineLocationOn } from 'react-icons/md';
+import { FaRegClock, FaTshirt } from 'react-icons/fa';
+
+interface IMovieData {
```
Comment (lysackm) [inline src/pages/Goosies/Goosies.tsx]: I dont see this used in this class, copy paste error?

File: src/components/EventTeaser/EventTeaser.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import { PropsWithChildren } from "react";
+
+interface IEventTeaser {
+    title: string;
+    image: string;
+    to?: string;
+}
+
+export const EventTeaser = ({title, image, to, children}: PropsWithChildren<IEventTeaser>) => {
+    return (
```
Comment (lysackm) [inline src/components/EventTeaser/EventTeaser.tsx:11]: Test on mobile and desktop

File: package.json [hunk 4]
```diff
@@ -20,6 +20,7 @@
     "history": "^5.3.0",
     "iframe-resizer-react": "^1.1.0",
     "mobx-react": "^9.0.0",
+    "randombytes": "^2.1.0",
```
Comment (lysackm) [inline package.json:23]: Make sure this package is necessary. I dont think its sketchy but we should try to not import every package under the sun.

File: src/pages/Goosies/Goosies.tsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+            Ready for the "BEST" AWARD show, for the Computer Science community by the Computer Science 
+            community? Come join us for this end-of-term event for lots of giggles and fun!
+            </Text>
+            <br></br>
+            <Text>
+            It's your chance to meet lots of new people and build connections. Have fun with us before the term ends!
+            </Text>
+            <br></br>
+            <Text>
+            Registration is required, so we know how many people to expect so we can serve you well!
```
Comment (lysackm) [inline src/pages/Goosies/Goosies.tsx:23]: add an and between expect and so

`people to expect and so that we can serve you well!`

**Review by lysackm — CHANGES_REQUESTED** [review]

File: src/components/EventTeaser/EventTeaser.tsx (hunk 3 above)
Comment (travisfriesen) [reply src/components/EventTeaser/EventTeaser.tsx:11]: tested on mobile and desktop, looks as intended

File: package-lock.json
Comment (lysackm) [inline package-lock.json:1]: If you removed the package, then there shouldnt be a diff in package-lock.json. I think  you need to run yarn install or yarn update. If that doesnt work, you can delete node-modules and do a fresh install, but that could take a while

File: src/pages/Goosies/Goosies.tsx [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+            Ready for the "BEST" AWARD show, for the Computer Science community by the Computer Science 
+            community? Come join us for this end-of-term event for lots of giggles and fun!
+            </Text>
+            <br></br>
+            <Text>
+            It's your chance to meet lots of new people and build connections. Have fun with us before the term ends!
+            </Text>
+            <br></br>
+            <Text>
+            Registration is required, so we know how many people to expect and so we can serve you well!
```
Comment (lysackm) [inline src/pages/Goosies/Goosies.tsx:23]:
```suggestion
            Registration is required so we know how many people to expect and we can be prepared!
```

File: src/pages/Goosies/Goosies.tsx [hunk 7]
```diff
@@ -0,0 +1,49 @@
+import { Text, HStack, Box, Icon, Button, Link } from "@chakra-ui/react";
+import { PageLayout } from "../../containers/PageLayout/PageLayout";
+import { EventTeaser } from "../../components/EventTeaser/EventTeaser";
+import { MdOutlineLocationOn } from 'react-icons/md';
+import { FaRegClock, FaTshirt } from 'react-icons/fa';
+
+export const Goosies = () => {
+        
+
```
Comment (lysackm) [inline src/pages/Goosies/Goosies.tsx:9]:
```suggestion
```

**Pull request merged**